root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -2240,6 +2240,21 @@ const testData = [
     expected_relationship_type: ['downloadfree', 'streamingfree'],
             expected_clean_url: 'http://www.jamendo.com/album/56372',
   },
+  // Prog Archives
```

Comment:
Typo from copy/paste:
```suggestion
  // Jazz Music Archives
```